src/main.ts

```diff
@@ -18,25 +20,22 @@ export function* main(): Sequence {
     inject: '<script src="http://localhost:4004/harness.js"></script>'
   }, (server) => {
     let address = server.address() as AddressInfo;
-    console.log(`-> proxy server listening on port ${address.port}`);
+    log.info(`-> proxy server listening on port ${address.port}`);
   }));
 
   // accept commands from the outside world (CLI, UI, etc...)
   fork(createServer(4000, commandServer, server => {
     let address = server.address() as AddressInfo;
-    console.log(`-> listening for commands on port ${address.port}`);
+    log.info(`-> listening for commands on port ${address.port}`);
   }));
 
-  fork(agentServer(4004, server => {
-    let address = server.address() as AddressInfo;
-    console.log(`-> agent server running on ${address.port}`);
-  }));
+  fork(agentServer(4004));
```

Comment:
I think we'll need to preserve the ability to wait for the server to start (though a callback isn't ideal for this anyway), but maybe it's okay to merge this as is.